docs/guides/first-app.mdx

```diff
@@ -330,8 +330,8 @@ You can also [touch and delete] relationships, but those are not as immediately
   <TabItem value="shell">
 
 ```shell
-zed relationship create posts:1 writer user:emilia
-zed relationship create posts:1 reader user:beatrice
+zed relationship create {{ tenant }}/post:1 writer {{ tenant }}/user:emilia
```

Comment:
In order for escaping to work, the code needs to be inside a `SampleCodeBlock`, like so:

```
<SampleCodeBlock lang="shell">{`zed context set blog {{ endpoint }} {{ token }}`}</SampleCodeBlock>
```

Reply:
As opposed to triple backtics?

Reply:
Yes. The component is required for us to process the templating

Reply:
I made a change - I'm not sure if I got the multiline syntax correct though. Can you take a look?

Reply:
Looks good! You can verify here: https://docs-git-fork-reify-tanner-stirrat-patch-1-authzed.vercel.app/guides/first-app

If it looks good to you as well, I'll approve

Reply:
Yep, looks good to me. Thanks!